2bttns/2bttns
# production setup

Live manual deployment to Vercel: https://2bttns-prod-test.vercel.app/

- Prod script setup
- .env.test overrides for testing
- Use workspace typescript SDK in vscode
- Linting fixes
- Production build setup
- Vercel deployment updates
- Fix /play URLSearchParams serversideprops check for production
- Update test db port so it doesn't conflict with dev db
- Use env.local in example app to config 2bttns
- move utils/2bttns.tsx out of pages for example app
- Update docs
